### ISSUE TITLE
eth_getBlockByHash response should include uncles and txs

### ISSUE THREAD

**Issue by njgheorghita** [ethereum/trin#279]
To be compliant with the eth json-rpc api spec, the [object returned from `eth_getBlockByHash` request](https://eth.wiki/json-rpc/API#eth_getblockbyhash) should include a list of uncle hashes and transaction hashes. Also, support for the `full_transactions` param would be nice to have.

**Comment by njgheorghita** [comment]
closing since we no longer support this endpoint

**Comment by carver** [comment]
> we no longer support this endpoint

Hm, but we should eventually. I would say support for `eth_getBlockByHash` is what it would mean to be ready for Devconnect.

**Comment by carver** [comment]
Depends on #885
